### DISCUSSION TITLE
Why are invitations not supported for SAML SSO?

### DISCUSSION THREAD

**Discussion by austintackaberry** [boxyhq/saas-starter-kit#1757] [Q&A]
I noticed that the invitations only seem to be relevant for non-SAML SSO login. Why is that? Is it because you expect SAML SSO invitations to be handled elsewhere?

**Comment by AdamDudley** [comment]
SAML SSO is managed by the customer. So on their own infra, often they have their own apps that provision apps to employees. The point is to let the customer do their thing with 100% control from their own IT dept.
